biofefi/pages/3_Data_Visualisation.py

```diff
@@ -109,7 +109,7 @@
         exec_opt.random_state,
         data_analysis_plot_dir,
         plot_opt,
-        exec_opt.normalization,
+        exec_opt.normalisation,
```

Comment:
```suggestion
        exec_opt.normalization,
```
backwards compatibility